common/artifacts/000_wc_artifacts.txt

```diff
@@ -568,6 +625,208 @@ runeblade_unholy_1 = {
 	slot = weapon
 }
 
+### One-handed weapon ###
+
+# Legendary
+felomelorn = {
+	quality = 5
+	flags = { combat_weapon sword }
+
+	active = {
+		is_adult = yes
+		OR = {
+			NOT = { has_class_trait_trigger = yes }
+			is_physical_class_trigger = yes
+			is_combined_class_trigger = yes
+		}
+		religion = cult_of_sunwell
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	diplomacy = 1
+	learning = 2
+	combat_rating = 10
+
+	monthly_character_prestige = 0.8
+	monthly_character_piety = 0.35
+	
+	general_opinion	= 5
+	cult_of_sunwell_opinion = 5
+
+	picture = "GFX_felomelorn"
+	slot = weapon
+}
+sulfuras = {
+	quality = 5
+	flags = { combat_weapon axe }
+
+	active = {
+		is_adult = yes
+		OR = {
+			NOT = { has_class_trait_trigger = yes }
+			is_physical_class_trigger = yes
+			is_combined_class_trigger = yes
+		}
+		OR = {
+			trait = giant
+			trait = strong
+			trait = robust
+			is_large_humanoid_trigger = yes
+		}
+		combat_rating = 50
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	martial = 2
+	intrigue = 1
+	learning = 1
+
+	health = 0.5
+	combat_rating = 25
+
+	monthly_character_prestige = 1.5
+#	monthly_character_piety = 0.4
+
+	picture = "GFX_sulfuras"
+	slot = weapon
+}
+hammer_of_the_twilight = {
+	quality = 5
+	flags = { combat_weapon hammer }
+
+	active = {
+		is_adult = yes
+		OR = {
+			NOT = { has_class_trait_trigger = yes }
+			is_magical_class_trigger = yes
+			is_combined_class_trigger = yes
+		}
```

Comment:
These conditions are strange. Like... characters without class can use it... but warriors can't.
I would either remove class conditions at all or limit it to...
```
OR = {
	is_magical_class_trigger = yes
	is_combined_class_trigger = yes
}
```
...mages and combined classes.

Reply:
i think, that's the first artifact where i tried to use these conditions. deleted the NOT condition now